catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveAllCommandTest.java

```diff
@@ -110,7 +188,9 @@ public void testExecuteWithSubject() throws Exception {
      */
     @Test
     public void testExecuteWithSubjectWithCache() throws Exception {
-        // given
+
+        int numCFCalls = 1;
+
```

Comment:
I seem to remember saying something like that on one of your PRs. I wouldn't want to be a hypocrite now :)